src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/WhyCommand/WhyCommandRunner.cs

```diff
@@ -89,6 +99,9 @@ public static int ExecuteCommand(WhyCommandArgs whyCommandArgs)
             return ExitCodes.Success;
         }
 
+        /// <summary>
+        /// Validates that the input 'PATH' argument is a valid path to a directory, solution file or project file.
```

Comment:
nitpick, writing PATH in capital letters immediately makes me think of the PATH environment variable. I'd write this comment as lowercase, to match the casing the parameter actually uses, especially since C# symbols are case sensitive

Reply:
That makes sense. We're also using lowercase 'path' everywhere else, so I've changed this one too